src/api/test/unit/package_remove_test.rb

```diff
@@ -0,0 +1,119 @@
+# Testing the things that should and shouldn't happen when you remove a Package
+require File.expand_path(File.dirname(__FILE__) + "/..") + "/test_helper"
+
+class PackageRemoveTest < ActiveSupport::TestCase
+
+  fixtures :all
+
+  def setup
```

Comment:
If you run a test alone the backend writing is disabled by default causing unexpected errors.